libs/vkd3d/resource.c

```diff
@@ -3881,20 +3903,28 @@ static void vkd3d_create_texture_srv(struct d3d12_desc *descriptor,
                 key.u.texture.view_type = VK_IMAGE_VIEW_TYPE_2D;
                 key.u.texture.miplevel_idx = desc->Texture2D.MostDetailedMip;
                 key.u.texture.miplevel_count = desc->Texture2D.MipLevels;
+                key.u.texture.miplevel_clamp = desc->Texture2D.ResourceMinLODClamp;
                 key.u.texture.layer_count = 1;
                 key.u.texture.aspect_mask = vk_image_aspect_flags_from_d3d12(resource->format, desc->Texture2D.PlaneSlice);
                 if (desc->Texture2D.ResourceMinLODClamp)
-                    FIXME_ONCE("Unhandled min LOD clamp %.8e.\n", desc->Texture2D.ResourceMinLODClamp);
+                {
+                    FIXME_ONCE("Wrong handling of min LOD clamp %.8e. Falling back to view clamp.\n",
```

Comment:
If we want to print a warning, we should do so inside `vkd3d_create_texture_view` to avoid duplication.

Reply:
Fixed.